Core/nbproject/project.properties

```diff
@@ -118,7 +118,7 @@ file.reference.StixLib.jar=release\\modules\\ext\\StixLib.jar
 file.reference.threetenbp-1.3.3.jar=release\\modules\\ext\\threetenbp-1.3.3.jar
 file.reference.webp-imageio-sejda-0.1.0.jar=release\\modules\\ext\\webp-imageio-sejda-0.1.0.jar
 file.reference.xmpcore-5.1.3.jar=release\\modules\\ext\\xmpcore-5.1.3.jar
-file.reference.YaraJNIWrapper.jar=release\\modules\\ext\\YaraJNIWrapper.jar
+file.reference.YaraJNIWrapper.jar=release/modules/ext/YaraJNIWrapper.jar
```

Comment:
Is this change correct?

Reply:
That change was some Netbeans magic that I decided not to fight. Apparently both styles of path separators are valid.